src/renderer/notification-handler.ts

```diff
@@ -64,8 +64,8 @@ export default class NotificationHandler {
      * Initializes / resets the notification positional values
      */
     public setupNotificationPosition() {
-        // This feature only applies to windows
-        if (isMac || isLinux || !app.isReady()) {
+        // This feature only applies to windows & mac
+        if (isLinux || !app.isReady()) {
```

Comment:
the code you've written should actually support linux as well right?

Reply:
Sure, will update